### DISCUSSION TITLE
need to add extension from cmd

### DISCUSSION THREAD

**Discussion by aberesnev94** [MicrosoftEdge/WebView2Feedback#4490] [Q&A] [answered]
Hi, i need to load extension to app with msedgewebview2. Does msedgewebview2 support --load-extension flag as chrome and msedge or it's support any other options except add code to app directly?

**Comment by victorhuangwq** [comment, marked as answer]
Here's the API for browser extensions:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?tabs=dotnetcsharp#browser-extensions
Example code here:
https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/WebView2APISample/ScenarioExtensionsManagement.cpp

**Reply by aberesnev94** [reply]
Do i have any other options? I don't have direct access to msedgewebview2

**Reply by victorhuangwq** [reply]
As far as I know, there isn't
